test/module/irohad/simulator/simulator_test.cpp

```diff
@@ -286,9 +286,9 @@ TEST_F(SimulatorTest, RightNumberOfFailedTxs) {
           .transactions(std::vector<shared_model::proto::Transaction>{tx})
           .build());
   auto tx_errors = iroha::validation::TransactionsErrors{
-      std::make_pair(validation::CommandError{"SomeCommand", "SomeError", true},
+      std::make_pair(validation::CommandError{"SomeCommand", 1, true},
```

Comment:
maybe add some number variable to avoid magic number here and upwards in this PR?

Reply:
It's covered in IR-1816 as well